views/default/page/elements/topbar_wrapper.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Elgg topbar wrapper
+ * Check if the user is logged in and display a topbar
+ * @since 1.10 
+ */
+if (!elgg_is_logged_in()) {
+	return true;
```

Comment:
Just `return;`. We shouldn't give the impression that return values mean anything.